WooCommerce/Classes/System/DefaultFeatureFlagService.swift

```diff
@@ -10,7 +10,7 @@ struct DefaultFeatureFlagService: FeatureFlagService {
         case .cardPresentPayments:
             return buildConfig == .localDeveloper || buildConfig == .alpha
         case .largeTitles:
-            return buildConfig == .localDeveloper || buildConfig == .alpha
+            return true
```

Comment:
I wonder if we should add this as a beta feature, to give testers the ability to report any kind of weird UI issue before making it available worldwide, what do you think?

Reply:
by beta feature, did you mean to put it behind a feature toggle in Settings > Experimental Features? if so, I'm not so sure how we'd announce this feature like the WIP banner we have the Products tab 🤔 

I was thinking we could enable large titles for the upcoming 6.2 TestFlight beta build, encouraging beta testers to report any UI issues with large titles to us (I'll update the release notes to reflect this point). If we receive a lot of issues that aren't easy to solve, we can turn off the feature flag before we publish 6.2. What do you think?

Reply:
> by beta feature, did you mean to put it behind a feature toggle in Settings > Experimental Features? if so, I'm not so sure how we'd announce this feature like the WIP banner we have the Products tab

Yeah, I meant that and I'm also not sure the best way to announce it 😅! but something inside me also tells me that shipping these UX glitches is not something that the woo app should be known for :-/

What do you think?

cc @woocommerce/mobile-designers @shiki

Reply:
Hi! I have no strong opinion on this. What we can probably consider is the effort it takes to maintain the experimental feature setting plus the announcement. And also consider how confident we are with this change. If we're fairly confident that nothing bad's gonna happen, then it's probably best to not spend time implementing the experimental feature setting. 

I'll leave it to you and @woocommerce/mobile-designers to decide what's best. 🙂

Reply:
In the spike p2 post, @adamzelinski had tested the installable build and gave a green light on shipping this feature (I think @Garance91540 had issues with installable builds). I just generated another installable build (#18914) if any of @woocommerce/mobile-designers would like to play with latest build with large titles!

If we don't go with the workaround, it requires significant design changes on the Dashboard and Orders tab to have another way to achieve the current top tab bar selection. I think we want to implement the large titles for these reasons:
- It's live in Android for a few releases
- The upcoming home screen update design uses the white navigation bar / large title design p91TBi-3i1-p2